src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheck.java

```diff
@@ -748,8 +748,14 @@ private static boolean isStandAloneIncrementOrDecrement(DetailAST identAst) {
      * @return true if variable nested in exprAst is used
      */
     private static boolean isIncrementOrDecrementVariableUsed(DetailAST exprAst) {
-        return TokenUtil.isOfType(exprAst.getParent(),
-                TokenTypes.ELIST, TokenTypes.INDEX_OP, TokenTypes.ASSIGN)
+        final int[] variableUsedTypes = {
```

Comment:
Please put this array to class field to be created ones, and not on each method call

Reply:
done